python/py-zmq/Portfile

```diff
@@ -59,11 +59,15 @@ if {${name} ne ${subport}} {
         livecheck.regex     {"filename":"pyzmq-(20.[0-9.]+).tar.gz",}
 
     } else {
+        python.pep517       yes
+        depends_build-append \
+                            port:py${python.version}-setuptools_scm
+        depends_lib-append \
+                            port:py${python.version}-packaging \
+                            port:py${python.version}-cffi \
+                            port:py${python.version}-cython
```

Comment:
I can take a look at some point myself but I doubt the cython is also a runtime dependency. Typically it's only a build dependency..

Reply:
I checked other ports before committing this. I saw it both ways quite a bit, so left it as a lib dependency to be on the safe side. Looked at the zmq source code as well, but as I haven't used cython myself then I'm not exactly sure if the python code actually requires the cython imports during runtime as well or not.

Reply:
it actually lists
  "cython>=3.0.0; implementation_name == 'cpython'",```
so that will not work currently as our `py-cython` port has not been upgrade yet to the latest upstream version; see also: https://trac.macports.org/ticket/68929

Reply:
however, upstream also says in the README:
```
Unless you specifically want to develop PyZMQ, we recommend downloading the PyZMQ source code or wheels from [PyPI](https://pypi.io/project/pyzmq/), or install with conda.

You can also get the latest source code from our GitHub repository, but building from the repository will require that you install recent Cython.
```

that implies that Cython isn't required if you get the source code from PyPI -which we do- so perhaps try to remove the dependency completely and see if it works (please make sure to uninstall or at least deactivate py-cython to make sure it doesn't opportunistically used if when it's installed on your system.

Reply:
Seems to work without cython. You can check if you're happy with the added patch.